### PR TITLE
[python3/en] Add single inheritance section

### PR DESCRIPTION
**Pull request by kaymmm**
Addresses #2808 by adding a new section to [python3/en] addressing single inheritance, and modifies multiple inheritance section accordingly.

- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: python3.html.markdown [hunk 1]
```diff
@@ -740,7 +740,101 @@ if __name__ == '__main__':
...
+    if type(sup) is Superhero:
+        print('I am a superhero')
+
+    # Get the Method Resolution search Order used by both getattr() and super()
+    # This attribute is dynamic and can be updated
+    print(Superhero.__mro__)    # => (<class '__main__.Superhero'>,
+                                # => <class 'human.Human'>, <class 'object'>)
+
+    # Calls parent method but uses its own class attribute
+    print(sup.get_species())    # => Superhero
```
Comment (Oire) [inline python3.html.markdown]: Superhuman?

File: python3.html.markdown [hunk 2]
```diff
@@ -740,7 +740,101 @@ if __name__ == '__main__':
...
+    def __init__(self, name, movie=False,
+                 superpowers=["super strength", "bulletproofing"]):
+
+        # add additional class attributes:
+        self.fictional = True
+        self.movie = movie
+        self.superpowers = superpowers
+
+        # The "super" function lets you access the parent class's methods
+        # that are overwritten by the child, in this case, the __init__ method.
```
Comment (Oire) [inline python3.html.markdown]: overridden?

File: python3.html.markdown [hunk 3]
```diff
@@ -766,20 +860,16 @@ if __name__ == '__main__':
     print(b.fly)
 
 # To take advantage of modularization by file you could place the classes above in their own files,
```
Comment (Oire) [inline python3.html.markdown]: You may delete this as you explain this in single inheritance part.

File: python3.html.markdown [hunk 4]
```diff
@@ -766,20 +860,16 @@ if __name__ == '__main__':
...
-# say, human.py and bat.py
+# say, superhero.py and bat.py
 
 # To import functions from other files use the following format
```
Comment (Oire) [inline python3.html.markdown]: ...and this, also.

File: python3.html.markdown [hunk 5]
```diff
@@ -802,19 +893,22 @@ if __name__ == '__main__':
...
-    if isinstance(sup, Human):
-        print('I am human')
+    if isinstance(sup, Superhero):
+        print('I am a superhero)
     if isinstance(sup, Bat):
         print('I am bat')
     if type(sup) is Batman:
         print('I am Batman')
 
     # Get the Method Resolution search Order used by both getattr() and super().
```
Comment (Oire) [inline python3.html.markdown]: This is also to be deleted, I believe.

**Comment by kaymmm** [comment]
I think it should be all set now (I agreed with all suggested changes). Happy to make additional changes as necessary!

**Comment by Oire** [comment]
@kaymmm, It's a pleasure to work with you. Thank you very much for fixing our bugs :)!

**Pull request merged**